db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+item_contents = ([{ name: "Chocolate Toast", description: "Dark chocolate on toast",
+        price: "3.00"}, { name: "Cake Toast", description: "Lemon poppyseed cake on toast", price: "4.00"},
+        { name: "Brownie Toast", description: "Delicious dark chocolate brownie on toast", price: "4.00"},
+        { name: "Cinnamon Sugar Toast", description: "Toast sprinkled with raw cane sugar, fresh ground
+        cinnamon, and a hint of sea salt", price: "2.50"}, { name: "Avacodo Toast", description: "Fresh
+        California avacado sliced over toast", price: "5.00"}, { name: "Egg Toast", description: "Choose
+        from fried or scrambled eggs!", price: "4.00"}, { name: "Dry Toast", description: "Nothing on it",
+        price: "2.00"}, { name: "Tuna Salad Toast", description: "With pickles,onions, and a hint of basil",
```

Comment:
Line is too long. [108/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.